Formula/inadyn.rb

```diff
@@ -0,0 +1,42 @@
+class Inadyn < Formula
+  desc "Dynamic DNS client with IPv4, IPv6, and SSL/TLS support"
+  homepage "http://troglobit.com/inadyn.html"
+  url "https://github.com/troglobit/inadyn/releases/download/v2.4/inadyn-2.4.tar.gz"
+  revision 0
+
+  head do
```

Comment:
Please remove the head section

Reply:
Right at this moment it only builds from head.

Why is this a request?  Hundreds of other formula have head, and there's no mention on not having head in the submission guide?

Reply:
One of the requirements is that new formulas should pass `brew audit --new-formula <formula>` and a formula containing a head section does not which the bot has already pointed out. This is mentioned in the Formula Cookbook  https://github.com/Homebrew/brew/blob/master/docs/Formula-Cookbook.md#basic-instructions. This didn't used to be enforced and has changed over time so the formulas that currently has it is generally older formulas that's been around for a while.

If the formula only builds from head at the moment this formula can not be accepted and we would have to wait until a stable release is out that builds from source.

Reply:
So I completely understand what you're saying, and I'm not wanting to argue so much as point out that what you're asking for doesn't match what the cookbook that you've linked to says. It says explicitly that if we want to allow people to grab a devel version we can include head or devel: https://github.com/Homebrew/brew/blob/master/docs/Formula-Cookbook.md#advanced-formula-tricks

There are others who are interested in testing the Homebrew formula while the next release is being prepared. If you don't want `head` blocks in the formula, what is the recommended path for sharing formula for testing purposes?

I have argument we should go one way or another, and I'm happy to comply. I'm just lacking instruction on how to make test releases available today. It seems like there's a new thought pattern on how this should be done (if I'm interpretting your responses correctly) but I can't figure it out from the documentation available today--which would appear to indicate that this is the correct path.